packages/backend/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
```

Comment:
same version as webchat?

Reply:
Will sync version using lerna and scripts.